.gitignore

```diff
@@ -16,6 +16,7 @@ packages/
 *.pdb
 tools/OutlineService/OutlineService/bin/*
 !tools/OutlineService/OutlineService/bin/*.exe
+tools/outline_proxy_controller/OutlineProxyControllerConfig.h
```

Comment:
You can also add a .gitignore to tools/outline_proxy_controller, so it doesn't break when moved (which we want to do)

Reply:
You mean the entire `tools/outline_proxy_controller` folder? Will it ignore all changes to the source code then?

Reply:
Nice catch, updated.